modules/imgproc/src/shapedescr.cpp

```diff
@@ -967,11 +993,46 @@ static Rect pointSetBoundingRect( const Mat& points )
                 minval2 = v_min(ptXY, minval2);
                 maxval2 = v_max(ptXY, maxval2);
             }
-            xmin = min(xmin, cvFloor(minval2.get0()));
-            xmax = max(xmax, cvFloor(maxval2.get0()));
-            ymin = min(ymin, cvFloor(v_reinterpret_as_f32(v_expand_high(v_reinterpret_as_u32(minval2))).get0()));
-            ymax = max(ymax, cvFloor(v_reinterpret_as_f32(v_expand_high(v_reinterpret_as_u32(maxval2))).get0()));
+            xmin = min(xmin, cvFloor(v_get0(minval2)));
+            xmax = max(xmax, cvFloor(v_get0(maxval2)));
+            ymin = min(ymin, cvFloor(v_get0(v_reinterpret_as_f32(v_expand_high(v_reinterpret_as_u32(minval2))))));
+            ymax = max(ymax, cvFloor(v_get0(v_reinterpret_as_f32(v_expand_high(v_reinterpret_as_u32(maxval2))))));
         }
+    #else // #if (CV_SIMD_WIDTH > 16 && !CV_SIMD) equal to "#elif CV_SIMD_SCALABLE"
+
+        if( i < npoints )
```

Comment:
I think it would be better to extract a function or two here to avoid duplication of code.

Reply:
Done